windows_setup.sh

```diff
@@ -48,24 +57,27 @@ else
     # This is due to opaque pointer types. Scroll down to "Version Support" here: https://llvm.org/docs/OpaquePointers.html
     # All WinLibs versions and download links: https://winlibs.com/
     if [ $small = yes ]; then
-        # A special small version of mingw64 that comes with the repo, for people with slow internet.
-        # See .github/workflows/windows.yml for code that checks the contents of the zip so that you can trust it.
-        # TODO: this is still LLVM 14
-        url=https://akuli.github.io/mingw64-small.zip
-        filename=mingw64-small.zip
-        sha=4d858bd22f084ae362ee6a22a52c2c5b5281d996f96693984a31336873b92686
+        # User has slow internet and doesn't want to download the whole mingw64 (about 1GB).
+        # Instead, download a release of Jou (about 50MB), and extract mingw and Jou compiler from there.
```

Comment:
These are wrong and will get outdated (more wrong) easily

```suggestion
        # User has slow internet and doesn't want to download the whole mingw64.
        # Instead, download a release of Jou, and extract mingw and Jou compiler from there.
```